src/Symfony/Component/Uid/UlidFactory.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Uid;
+
+class UlidFactory
+{
+    /**
+     * @var string|null
+     */
+    private $defaultTimestamp;
```

Comment:
same comment as for UuidFactory, not sure this maps to a use case, and it looks like a footgun to me.